docs/commerce/checkout/configure_checkout.md

```diff
@@ -0,0 +1,237 @@
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\SummaryStepController::renderStepView
+                        label: 'Payment & summary'
+                        translation_domain: checkout
+                complete:
+                    from: summarized
+                    to: completed
+                    metadata:
+                        controller: Ibexa\Bundle\Checkout\Controller\CheckoutStep\CompleteStepController::renderCompleteView
+                        label: 'Order confirmation'
+                        translation_domain: checkout
+```
+
+## Configure shipping and payment methods
+
+You can define the shipping and payment methods.
+Under `ibexa.repositories.<repository_name>.checkout`, create entries that resemble 
+the following example:
+
+``` yaml 
+ibexa:
+    repositories:
+        <repository_name>:
+            checkout:
+                workflow: <your_workflow_identifier> #optional, "ibexa_checkout" is used by default
+                shipping_methods:
+                    courier:
+                        name: "Courier"
+                        translation_domain: "checkout" #optional, all shipping/payment methods use this one by default
+                    parcel_machine:
+                        name: "Parcel machine"
+                    self_pickup:
+                        name: "Self pickup"
+                    express_delivery:
+                        name: "Express delivery"
+                payment_methods:
+                    credit_card:
+                        name: "Credit card"
+                    paypal:
+                        name: "Paypal"
+                    money_transfer:
+                        name: "Money transfer"
+```
+
+## Shipping and billing address field format configuration 
+
+In your implementation, you may need to create custom format configurations 
+for the shipping or billing address fields, for example, to use different address 
+formats based on the buyer's geographical location.
+
+Field formats for the billing and shipping addresses comply with the [FieldType Address](../../content_management/field_types/field_type_reference/addressfield.md#formats) specification and can be controlled with the `billing_address_format` and `shipping_address_format` flags, respectively.
+They fallback to `billing` and `shipping` predefined formats by default:
+
+- `billing` is sourced from the `corporate-accounts` bundle 
+- `shipping` is part of the `checkout` bundle's default configuration 
+
+To modify address formats you can either override the existing formats or create 
+custom ones.
+
+### Override existing Address Field Type formats 
+
+To override the existing `billing` and `shipping` address Field Type formats, edit 
+the `../vendor/ibexa/checkout/src/bundle/Resources/config/fieldtype_address.yaml` 
+file, and modify the address format, for example, by adding a tax identifier 
+to the `billing` address:
```

Comment:
Again, generally speaking **the customer shouldn't edit any of our source code to override configuration**. If someone wants to have their own format, they should provide it according to Address Fieldtype doc and reference it's key using `billing_address_format` / `shipping_address_format`.